src/types/automation/CloudTelevision.h

```diff
@@ -0,0 +1,219 @@
+// which covers the main part of ArduinoCloudThing.
+// The terms of this license can be found at:
+// https://www.gnu.org/licenses/gpl-3.0.en.html
+//
+// You can be released from the requirements of the above licenses by purchasing
+// a commercial license. Buying such a license is mandatory if you want to modify or
+// otherwise use the software for commercial activities involving the Arduino
+// software without disclosing the source code of your own applications. To purchase
+// a commercial license, send an email to license@arduino.cc.
+//
+
+#ifndef CLOUDTELEVISION_H_
+#define CLOUDTELEVISION_H_
+
+/******************************************************************************
+   INCLUDE
+ ******************************************************************************/
+
+#include <Arduino.h>
+#include "../../ArduinoCloudProperty.h"
+
+/******************************************************************************
+   ENUM
+ ******************************************************************************/
+enum class PlaybackCommands : u_int8_t {
+  FastForward   = 0,
+  Next          = 1,
+  Pause         = 2,
+  Play          = 3,
+  Previous      = 4,
+  Rewind        = 5,
+  StartOver     = 6,
+  Stop          = 7,
+  None          = 255
+};
+enum class InputValue : u_int8_t {
+  AUX1          = 0,
+  AUX2          = 1,
+  AUX3          = 2,
+  AUX4          = 3,
+  AUX5          = 4,
+  AUX6          = 5,
+  AUX7          = 6,
+  BLURAY        = 7,
+  CABLE         = 8,
+  CD            = 9,
+  COAX1         = 10,
+  COAX2         = 11,
+  COMPOSITE1    = 12,
+  DVD           = 13,
+  GAME          = 14,
+  HDRADIO       = 15,
+  HDMI1         = 16,
+  HDMI2         = 17,
+  HDMI3         = 18,
+  HDMI4         = 19,
+  HDMI5         = 20,
+  HDMI6         = 21,
+  HDMI7         = 22,
+  HDMI8         = 23,
+  HDMI9         = 24,
+  HDMI10        = 25,
+  HDMIARC       = 26,
+  INPUT1        = 27,
+  INPUT2        = 28,
+  INPUT3        = 29,
+  INPUT4        = 30,
+  INPUT5        = 31,
+  INPUT6        = 32,
+  INPUT7        = 33,
+  INPUT8        = 34,
+  INPUT9        = 35,
+  INPUT10       = 36,
+  IPOD          = 37,
+  LINE1         = 38,
+  LINE2         = 39,
+  LINE3         = 40,
+  LINE4         = 41,
+  LINE5         = 42,
+  LINE6         = 43,
+  LINE7         = 44,
+  MEDIAPLAYER   = 45,
+  OPTICAL1      = 46,
+  OPTICAL2      = 47,
+  PHONO         = 48,
+  PLAYSTATION   = 49,
+  PLAYSTATION3  = 50,
+  PLAYSTATION4  = 51,
+  SATELLITE     = 52,
+  SMARTCAST     = 53,
+  TUNER         = 54,
+  TV            = 55,
+  USBDAC        = 56,
+  VIDEO1        = 57,
+  VIDEO2        = 58,
+  VIDEO3        = 59,
+  XBOX          = 60
+};
+
+/******************************************************************************
+   CLASS DECLARATION
+ ******************************************************************************/
+
+class Television {
+  public:
+    bool                swi;
+    u_int8_t            vol;
```

Comment:
Not sure where `u_int8_t` is defined but the standard definition of those primitive types within `<stdint.h>` is `uint8_t`. Please adjust here and on all other locations within the code.

Reply:
Done